tests/units/typing/tensor/test_tensor.py

```diff
@@ -52,6 +51,60 @@ def test_unwrap():
     assert (ndarray == np.zeros((3, 224, 224))).all()
 
 
+def test_ellipsis_in_shape():
+    # ellipsis in the end, two extra dimensions needed
+    tensor = parse_obj_as(NdArray[3, ...], np.zeros((3, 128, 224)))
```

Comment:
can you make this test for torch tensor and tensorflow as well ?

Reply:
should I parametrize it in this file or add tests into their separate test files?

Reply:
you can parametrize for TorchTensor and NDarray. For tensorflow tensor you can add one or two test but not all of them.

Ideally we would only need to validate this feature at a unit test level and kind of be sure that it is working for all of the backend. But I think it is tough to do the unit test properly there